Apps/Sandcastle/gallery/Terrain.html

```diff
@@ -102,6 +110,12 @@
                 viewer.terrainProvider = ellipsoidProvider;
               },
             },
+            {
+              text: "CustomHeightmapTerrainProvider",
```

Comment:
I feel like this deserves its own Sandcastle called "Procedural Terrain" where the FBM and sine-wave are two dropdowns.

Reply:
I created a `Procedural Terrain` Sandcastle with noise and sine, and put the sine in the `Terrain` sandcastle as well